server/utils/applications/utils.test.ts

```diff
@@ -706,7 +706,7 @@ describe('utils', () => {
       const application = applicationFactory.build({ id: 'an-application-id', status: 'awaitingPlacement' })
 
       expect(actionsCell(application)).toEqual({
-        html: '<ul class="govuk-list"><li><a href="/applications/an-application-id/withdrawals/new"  >Withdraw</a></li><li><a href="/placement-applications"  >Request for placement</a></li></ul>',
+        html: '<ul class="govuk-list"><li><a href="/applications/an-application-id/withdrawals/new"  >Withdraw</a></li><li><a href="/placement-applications?id=an-application-id"  >Request for placement</a></li></ul>',
```

Comment:
Given this test was passing before I think we'll need more confidence that the change has fixed the issue in question. Could you write an integration test for this functionality?   
IE:  
- Given I have an application in an 'awaiting placement' state 
- When I click 'Request for placement' 
- Then I am taken to the 'Create a request for placement' form